.env.example

```diff
@@ -60,7 +59,3 @@ REQUEST_THROTTLE_MS=5000
 REQUEST_TIMEOUT_MS=5000
 RESIZING_SERVICE=gemini
 RESOLVER_TIMEOUT_MS=0
-
-ENABLE_SCHEMA_STITCHING=true
-ENABLE_GRAVQL_ONLY_STITCHING=true
-ENABLE_KAWS_STITCHING=true
```

Comment:
ENABLE_SCHEMA_STITCHING is now inverses: `DISABLE_SCHEMA_STITCHING`

Kaws & GravQL are now stitched by default.

Reply:
Should there be a commented out example of how to disable it so in a moment of crisis people can find it here?